lib/data/datasource/local/DAOs/database.dart

```diff
@@ -36,4 +36,25 @@ class Database {
     var response = await moviesCollection.get();
     return response;
   }
+
+  Future<List<QuerySnapshot>> searchMovies({
+    required String keyword,
+  }) async {
+    List<QuerySnapshot> results = [];
+    results.add(
+      await _mainCollection
+          .doc(ServiceConstants.tabsCollections['document'])
+          .collection(ServiceConstants.tabsCollections['collection']!)
```

Comment:
you can add 'document' and 'collection' to a constant value

Reply:
Added